ArchiSteamFarm/ArchiWebHandler.cs

```diff
@@ -890,17 +889,9 @@ internal sealed class ArchiWebHandler : IDisposable {
 		}
 
 		internal async Task<string> GetTradeToken() {
```

Comment:
Remove old AWH way of getting token entirely, we never expect AH to fail, this routine should never be executed.